client/src/components/layout/ReviewEditButton.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { Link } from "react-router-dom"
+
+const ReviewEditButton = ({ user, review }) => {
+  return(
+    <>
+      { user.id === review.userId ? <Link to={{
+          pathname: "/edit-review",
+          state: {user: user, review: review }
+          }} className="edit-button">Edit Review</Link> : null }
```

Comment:
fun that you found this. Too bad we have to refactor

If you are redirecting to a ReviewEditForm, you should be making a useEffect fetch request for the review you need. And currentUser should come down via props.

Avoid passing state into Link tags unless you truly have no other option. This is not one of those time.